README.md

```diff
@@ -25,11 +25,12 @@ Go to the [documentation directory](docs/).
 
 ### Setup
 
-Make sure you have your [_$GOPATH_ environment variable set](https://golang.org/doc/code.html#GOPATH).
+Make sure you have your [_$GOPATH_ environment variable set](https://golang.org/doc/code.html#GOPATH). $GOPATH should be
+set to the terraform-provider-softlayer directory.
```

Comment:
No, the GOPATH should be set to any arbitrary directory, except the project directory.